src/main/java/org/hydev/mcpm/client/updater/UpdateInteractor.java

```diff
@@ -20,7 +17,11 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import static org.hydev.mcpm.client.updater.UpdateOutcome.State.*;
+import static org.hydev.mcpm.client.updater.UpdateOutcome.State.MISMATCHED;
+import static org.hydev.mcpm.client.updater.UpdateOutcome.State.NETWORK_ERROR;
+import static org.hydev.mcpm.client.updater.UpdateOutcome.State.NOT_INSTALLED;
+import static org.hydev.mcpm.client.updater.UpdateOutcome.State.UPDATED;
+import static org.hydev.mcpm.client.updater.UpdateOutcome.State.UP_TO_DATE;
```

Comment:
is there a warning for importing *

Reply:
I don't know. IntelliJ just cleaned this up on its own when Azalea hit the cleanup code button. Typically, check style doesn't like star imports, but it hasn't been complaining lately.